projects/fal/src/fal/app.py

```
@@ -40,27 +40,26 @@ async def _call_any_fn(fn, *args, **kwargs):
         return fn(*args, **kwargs)
 
 
-async def open_isolate_channel(address: str) -> async_grpc.Channel:
-    _stack = AsyncExitStack()
-    channel = await _stack.enter_async_context(
-        async_grpc.insecure_channel(
-            address,
-            options=[
-                ("grpc.max_send_message_length", -1),
-                ("grpc.max_receive_message_length", -1),
-                ("grpc.min_reconnect_backoff_ms", 0),
-                ("grpc.max_reconnect_backoff_ms", 100),
-                ("grpc.dns_min_time_between_resolutions_ms", 100),
-            ],
-        )
+async def open_isolate_channel(address: str) -> async_grpc.Channel | None:
+    channel = async_grpc.insecure_channel(
+        address,
+        options=[
+            ("grpc.max_send_message_length", -1),
+            ("grpc.max_receive_message_length", -1),
+            ("grpc.min_reconnect_backoff_ms", 0),
+            ("grpc.max_reconnect_backoff_ms", 100),
+            ("grpc.dns_min_time_between_resolutions_ms", 100),
+        ],
     )
 
-    channel_status = channel.channel_ready()
     try:
+        channel_status = channel.channel_ready()
+
         await asyncio.wait_for(channel_status, timeout=1)
     except asyncio.TimeoutError:
-        await _stack.aclose()
-        raise Exception("Timed out trying to connect to local isolate")
```

we were killing the request instead of silently continuing